src/components/Facepile/Facepile.Props.ts

```diff
@@ -16,6 +16,9 @@ export interface IFacepileProps extends React.Props<Facepile> {
   /** Show add person button */
   showAddButton?: boolean;
 
+  /** Show persona details when only one face is present in the Facepile */
+  personaDetailsShown?: boolean;
```

Comment:
I'm a bit confused by this property. If true, will only show personal details in the personas, only if one face is present? That seems really proprietary.

I think a better approach would be don't do this by default, and if you really need this functionality allow any persona prop to be overridden for proprietary scenarios:

```ts
getPersonaProps?: (persona?: IFacepilePersona) => IPersonaProps;
```

Then mix that in the result into each persona. Then if you really want this feature, you can just provide the persona-specific override:

```tsx
<Facepile
  getPersonaProps: () => { hidePersonaDetails: true }
/>
```

Because today, it's just that feature. Tomorrow you'll want to tweak the persona size at different mobile stops. We'll end up eventually adding all the persona props here.